module/zfs/vdev_disk.c

```diff
@@ -173,8 +173,8 @@ vdev_elevator_switch(vdev_t *v, char *elevator)
 	strfree(argv[2]);
 #endif /* HAVE_ELEVATOR_CHANGE */
 	if (error)
-		printk("ZFS: Unable to set \"%s\" scheduler for %s (%s): %d\n",
-		    elevator, v->vdev_path, device, error);
+		printk(KERN_NOTICE "ZFS: Unable to set \"%s\" scheduler"
+		" for %s (%s): %d\n", elevator, v->vdev_path, device, error);
```

Comment:
nit: should be indented by 4 like the other `printk()`.  I'm a little surprised the style checker didn't catch this.